src/docs/self-hosted/index.mdx

```diff
@@ -12,7 +12,27 @@ Our recommendation is to download the [latest release of the self-hosted reposi
 
 When you run `./install.sh`, you have a choice to opt in or out of our monitoring. This monitoring is used for development and debugging purposes so that we're on top of issues you're facing, allowing us to provide a more seamless installation process. For more details please see the [section in the self-hosted README](https://github.com/getsentry/self-hosted#self-hosted-monitoring).
 
-Note that choosing whether to send errors or not will become mandatory beginning with the 22.10.0 release.
+Note that choosing whether to send errors or not will become mandatory beginning with the 22.11.0 release.
+
+### Self-hosted Beacon
+
+Self-hosted will periodically communicate with a remote beacon server. This is utilized for a couple of things, primarily:
```

Comment:
This is a configured option at install time, right? Right now this reads as a always-on feature whereas I think it is important to be clear it is opt-in.

Maybe something like 
```suggestion
If you opt-in to it, self-hosted Sentry will periodically communicate with a remote beacon server. This is utilized for a couple of things, primarily:
```